arangod/Replication2/StateMachines/Document/DocumentLogEntry.h

```diff
@@ -59,7 +61,9 @@ auto inspect(Inspector& f, OperationType& p) {
       OperationType::kCommit, "Commit",                          //
       OperationType::kAbort, "Abort",                            //
       OperationType::kAbortAllOngoingTrx, "AbortAllOngoingTrx",  //
-      OperationType::kIntermediateCommit, "IntermediateCommit"   //
+      OperationType::kIntermediateCommit, "IntermediateCommit",  //
+      OperationType::kCreateShard, "CreateShard",                //
+      OperationType::kDropShard, "DropShard"                     //
```

Comment:
Not sure if we need a `ModifyShard` operation as well? Someone should look into which properties of a shard can change 🤔

Reply:
I will cover that up. I didn't wanted to add `ModifyShard` in here until I know for sure what it is supposed to do.